### PR TITLE
fix: Set hitbox `debugColor` to yellow

### PR DESCRIPTION
**Pull request by spydon**
# Description

To easily be able to spot whether a it is a hitbox that is being drawn in `debugMode = true` it is now yellow instead of purple like the other debug boxes.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import 'package:flame/collisions.dart';
 import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
```
Comment (erickzanardo) [inline packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart:3]: Why import extensions here? 😅

**Review by erickzanardo — APPROVED** [review]
A question but approved

File: packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart (hunk 1 above)
Comment (spydon) [reply packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart:3]: It's instead of importing `dart:ui`, shouldn't really matter. :)

Comment (erickzanardo) [reply packages/flame/lib/src/collisions/hitboxes/shape_hitbox.dart:3]: Def not, was just curious haha

**Pull request merged**